fighthealthinsurance/followup_emails.py

```diff
@@ -55,7 +55,11 @@ def dosend(
             interested_pro.thankyou_email_sent = True
             interested_pro.save()
             return True
-        except:
+        except Exception as e:
+            # Log the error for debugging
+            import logging
+            logger = logging.getLogger(__name__)
```

Comment:
Don't need to do this here. Also we're already using loguru elsewhere in the code.